packages/mui-base/src/useMenu/useMenu.ts

```diff
@@ -14,6 +14,7 @@ import {
 } from './useMenu.types';
 import { EventHandlers } from '../utils';
 import useMenuChangeNotifiers from '../MenuUnstyled/useMenuChangeNotifiers';
+import { type MenuUnstyledContextType } from '../MenuUnstyled';
```

Comment:
Is this valid TypeScript?

Reply:
It seems like it is since it gets compiled. But I changed it to the conventional syntax in #36411. If we want to import multiple types then this won't work whereas `import type` will work. Thanks for pointing out.